### DISCUSSION TITLE
Welcome to ka-notifications Discussions!

### DISCUSSION THREAD

**Discussion by ejmurc** [eliasmurcray/ka-notifications#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Reginald-Gillespie** [comment]
It works 😎

**Reply by ejmurc** [reply]
Nice!

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/124937565/223256230-eae72453-4e34-45c2-91a2-547a89228ca1.png)
thats what it says

**Reply by Reginald-Gillespie** [reply]
Can you try accessing in incogneto?

**Reply by ejmurc** [reply]
It should not be doing that. Try accessing in incognito as WKoA said, or perhaps disable cache and reload

**Reply by ghost** [reply]
no, im not. why would I lol

it worked when i wasnt signed in lol

**Reply by ejmurc** [reply]
That is what we meant, it was probably just an issue with the Google Account. So everything works properly now?

**Reply by Reginald-Gillespie** [reply]
Yeah that's why I said try incogneto, because it would make you not signed in. I'm thinking maybe it's a restriction on your account blocking you from accessing other's files.

**Comment by ghost** [comment]
It just keeps saying I have 10 notifications now =/
Sometimes pops up again when no one said anything too.

**Reply by ejmurc** [reply]
I will check it out, it seems xacer has this problem as well.

**Reply by ejmurc** [reply]
Tracking now on #17

**Comment by ghost** [comment]
Notifications aren't even showing up anymore...

**Reply by ejmurc** [reply]
Any sort of error when you "Inspect Popup"?

**Reply by ghost** [reply]
```
ERROR [getFullUserProfile]:  Error in GraphQL getFullUserProfile call: Server responded  with status 403 and body "routes._validatePermission: [unauthorized error] Operation not found in safelist, hash = 3769880111, operation = getFullUserProfile"
(anonymous) @ background.js:1


```

**Reply by ejmurc** [reply]
Are you logged in? Also which extension version is this.

**Reply by ghost** [reply]
nvm, working again. just needed to update chrome.

**Reply by Reginald-Gillespie** [reply]
Polar recently added the offscreen API, which is only supported by newer chrome updates, so makes sense that it didn't install the latest extension until you updated chrome.

**Reply by ghost** [reply]
I think I got a notif on his subpage, then just stopped receiving them.

**Reply by ghost** [reply]
Once again, I'm getting infinite loading.

this is what pops up on the console
![image](https://user-images.githubusercontent.com/124937565/234392181-e9828c76-c808-48c3-8c19-ed37013234d3.png)

**Reply by ejmurc** [reply]
Check out the new update v1.3.11 it fixes this :)

Navigate to chrome://extensions >> Click Update on top left

**Comment by FakeCxrpss** [comment]
Would it be possible to add tabs in the extension for the community tab and the help request tab? I try to be as quick as possible with answering those and that would help a lot with that.

**Reply by ejmurc** [reply]
Are you asking for a filter for notifications, or are you referring to the Project Help unit for feedback across all projects on Khan Academy?

**Reply by FakeCxrpss** [reply]
I mean you can get the option to get to get notifications for all help request or questions/discussions tab

**Reply by ejmurc** [reply]
Are you referring to this? https://www.khanacademy.org/computing/computer-programming/projectfeedback

**Reply by FakeCxrpss** [reply]
I'm referring to that is this https://www.khanacademy.org/computing/computer-programming/d

**Reply by Reginald-Gillespie** [reply]
Tomato tomoto, I'd add both, backend is probably identical except just passing a differnet comment type
